changelog.json

```
@@ -1,5 +1,9 @@
 {
   "releases": {
+    "2.7.3-beta3": [
+      "[New] Add an alias to your repos and name them however you want! - #12000",
```

We typically use the issue number here unless there isn't one.
```suggestion
      "[New] Create aliases for repositories you want to be displayed differently in the repository - #7856",
```

---

Re the issue number: I think that's my fault because I didn't link the PR to the original issue, so the script didn't pick it up, sorry! 🤦 

Regarding the release note change:
> in the repository

Do you mean `in the repository list`? Otherwise it reads weird to me 🤔 😳

---

I did mean that! 😆